app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java

```diff
@@ -160,6 +166,35 @@ private String decideSessionTimeRange(Context context, Session session) {
                 DateUtil.getMinutes(displaySTime, displayETime));
     }
 
+    private PendingIntent createAlarmIntent() {
```

Comment:
Can you create `util/AlermUtil.java` and extract these logic to the util class? 🙇